### ISSUE TITLE
Improve structure of data import code

### ISSUE THREAD

**Issue by rasmuse** [demokratikollen/demokratikollen#14]
The code we are using to clean up and import data files from the parliament will soon grow into a big pile of spaghetti as we continue to add ad hoc solutions to all the funny errors they have in their files. (See my recent comment on #3 and commit 0994020108.)

I propose that we make something like a catalog of corrections that can be applied to these data files. Adding support for input data file then essentially becomes a matter of mapping a series of corrections to a file name. (Plus coming up with new corrections and adding them to the catalog, of course.)

This seems like a detail-oriented and unrewarding piece of work, so I assign myself to it.

**Comment by jeinarsson** [comment]
Sounds like a good plan.

I'm eagerly anticipating the day Riksdagen sorts out their API... any year now probably!

**Comment by rasmuse** [comment]
Data import code is now significantly improved. To setup the database we can now go to the application root, `cd ~/demokratikollen` and then

```
python import_data.py wipe
python import_data.py download data/urls.txt data/download
python import_data.py unpack data/download data/download
python import_data.py clean data/download data/cleaned --remove
python import_data.py execute data/cleaned
```

See also README.md in the `import_data` branch.

**Comment by rasmuse** [comment]
I am reopening this issue just to say I have added a subcommand `auto` to the data import script. To initialize the database from scratch, now you just have to do

```
cd ~/demokratikollen
python import_data.py auto data/urls.txt --wipe
```

I have also updated README.md and merged this into `develop` with fdea23482a1545deb199b.
